lib/saluki-io/src/deser/codec/dogstatsd/mod.rs

```diff
@@ -161,6 +161,16 @@ impl<TMI> DogstatsdCodec<TMI> {
         buf.advance(data.len() - remaining.len());
         Ok(1)
     }
+
+    fn decode_service_check<B: ReadIoBuffer>(
+        &mut self, buf: &mut B, events: &mut EventBuffer,
+    ) -> Result<usize, ParseError> {
+        let data = buf.chunk();
+        let (remaining, service_check) = parse_dogstatsd_service_check(data, &self.config)?;
+        events.push(saluki_event::Event::ServiceCheck(service_check));
```

Comment:
No need for this to be fully qualified since `Event` is already imported and in scope.